Bumping version number breaks gcode exports

**Steps to make the bug happen**
1. Change pom.xml <version> value.
2. Maven > package
4. See error

**What was supposed to happen**
Gcode is exporting the version of makelangelo.  This breaks all gcode tests every time the version is changed.

**Platform (please complete the following information):**
 - Version: 7.32.0

**Log file**
```
[ERROR] Failures: 
[ERROR]   SaveGCodeTest.saveMultiColor:38->verifySavedFile:61 iterable contents differ at index [0], expected: <;Generated with Makelangelo 7.32.0> but was: <;Generated with Makelangelo 7.32.1>
[ERROR]   SaveGCodeTest.saveTurtle:33->verifySavedFile:61 iterable contents differ at index [0], expected: <;Generated with Makelangelo 7.32.0> but was: <;Generated with Makelangelo 7.32.1>
```

**Additional context**
If there's a process to regenerate the gcode then please document it.

--- a/src/test/java/com/marginallyclever/makelangelo/plotter/plottercontrols/SaveGCodeTest.java
+++ b/src/test/java/com/marginallyclever/makelangelo/plotter/plottercontrols/SaveGCodeTest.java
@@ -96,7 +96,7 @@ class SaveGCodeTest {
 
     private List<String> splitAndFilterForTest(String fileContent) {
         return Arrays.stream(fileContent.split("\\r?\\n"))
-                .filter(line -> !line.matches("; 20.* at ..:.*") && !line.matches("Generated with.*"))
+                .filter(line -> !line.matches("; 20.* at ..:.*") && !line.matches(";Generated with.*"))
                 .collect(Collectors.toList());
     }
 }

--- a/src/test/resources/gcode/multi_shapes_ignatus2.gcode
+++ b/src/test/resources/gcode/multi_shapes_ignatus2.gcode
@@ -1,4 +1,4 @@
-;Generated with Makelangelo 7.32.0 (dxf-5a62518)
+;Generated with Makelangelo 1.0.0 (ignored-5a62518)
 ;FLAVOR:Marlin-polargraph
 ;MINX:-94.500
 ;MINY:-47.744

--- a/src/test/resources/gcode/multi_shapes_path-circle-line-rect.gcode
+++ b/src/test/resources/gcode/multi_shapes_path-circle-line-rect.gcode
@@ -1,4 +1,4 @@
-;Generated with Makelangelo 7.32.0 (dxf-5a62518)
+;Generated with Makelangelo 1.0.0 (ignored-5a62518)
 ;FLAVOR:Marlin-polargraph
 ;MINX:-92.030
 ;MINY:-133.650

--- a/src/test/resources/gcode/save_multi_colors-1.gcode
+++ b/src/test/resources/gcode/save_multi_colors-1.gcode
@@ -1,4 +1,4 @@
-;Generated with Makelangelo 7.32.0
+;Generated with Makelangelo 1.0.0 (ignored-5a62518)
 ;FLAVOR:Marlin-polargraph
 ;MINX:-15.000
 ;MINY:-7.000

--- a/src/test/resources/gcode/save_multi_colors-2.gcode
+++ b/src/test/resources/gcode/save_multi_colors-2.gcode
@@ -1,4 +1,4 @@
-;Generated with Makelangelo 7.32.0
+;Generated with Makelangelo 1.0.0 (ignored-5a62518)
 ;FLAVOR:Marlin-polargraph
 ;MINX:-15.000
 ;MINY:-7.000

--- a/src/test/resources/gcode/save_multi_colors.gcode
+++ b/src/test/resources/gcode/save_multi_colors.gcode
@@ -1,4 +1,4 @@
-;Generated with Makelangelo 7.32.0
+;Generated with Makelangelo 1.0.0 (ignored-5a62518)
 ;FLAVOR:Marlin-polargraph
 ;MINX:-15.000
 ;MINY:-7.000

--- a/src/test/resources/gcode/save_simple_move.gcode
+++ b/src/test/resources/gcode/save_simple_move.gcode
@@ -1,4 +1,4 @@
-;Generated with Makelangelo 7.32.0
+;Generated with Makelangelo 1.0.0 (ignored-5a62518)
 ;FLAVOR:Marlin-polargraph
 ;MINX:-15.000
 ;MINY:-7.000

